content/docs/configuration.md

```diff
@@ -563,14 +569,102 @@ jobs:
   trigger: release
   metadata:
     dist_git_branches:
-      - f32
+      - f35
 ```
 
-This config would update Fedora Rawhide and Fedora 32 dist-git branches.
+This config would update Fedora Rawhide and Fedora 35 dist-git branches.
 
-**tests**
+**koji\_build**
 
-See more about tests [here](http://packit.dev/testing-farm/).
+Trigger the build in
+[Fedora Koji](https://koji.fedoraproject.org/koji/) build system
+as a reaction to a new dist-git commit.
+A Packit config file needs to be in the dist-git repository
+to allow this job to be triggered.
+Packit loads the config from the newly pushed commit.
+
+The build is triggered only for commits with a spec-file change.
+
+There is no UI provided by Packit for the job,
+but it is visible across Fedora systems (e.g. via dist-git commit status)
+like a manually created Koji build and you can utilise
+[Fedora Notifications](https://apps.fedoraproject.org/notifications/about)
+to get informed about the builds.
+
+For Koji builds from upstream, see `production_build`.
+(The naming is not ideal, but we don't want to change this because of the backwards compatibility.)
+
+Supported triggers:
+
+* **commit** -- reacts to new commits to the specified branch (in dist-git)
+
+Required metadata:
+
+* **dist_git_branches** -- the name of the dist-git branch we want to build for when using **commit** trigger.
+  Aliases like `fedora-all`, `fedora-stable` or `fedora-development` are supported.
+
+Optional metadata:
+
+* **scratch** -- defaults to `false`, use to create scratch (test) builds
+  instead of the real production builds
+
+**Example**
+
+```yaml
+jobs:
+- job: koji_build
+  trigger: commit
+  metadata:
+    dist_git_branches:
+      - fedora-all
+      - epel-8
+```
+
+**bodhi\_update** (preview)
+
+Create a new update in
+[Fedora Bodhi](bodhi.fedoraproject.org) for successful
+Koji build.
+A Packit config file needs to be in the dist-git repository
+to allow this job to be triggered.
+Packit loads the config from the commit the build is triggered from.
+
+For now, the Bodhi update is created only for builds submitted by the Packit FAS user.
+(See `koji_build` job for more details on how to set this up.)
+This is just for the early stage of this job and
+we can easily turn off that filter.
+Let us know if you need this condition to be removed.
+
+There is no UI provided by Packit for the job,
+but it is visible across Fedora systems
+like a manually created Bodhi update and you can utilise
```

Comment:
```suggestion
like a manually created Bodhi update and you can utilize
```
didn't know it can be spelt both with 's' and 'z'

Reply:
so I guess this is not a suggestion, just me poking spell checker :D

Reply:
British vs American spelling?..;)

Reply:
😁
ahaaaaaaa